api/v1alpha7/types.go

```diff
@@ -163,6 +163,15 @@ type RootVolume struct {
 	AvailabilityZone string `json:"availabilityZone,omitempty"`
 }
 
+type DataVolume struct {
+	NameSuffix       string `json:"nameSuffix"`
+	Size             int    `json:"diskSize,omitempty"`
+	VolumeType       string `json:"volumeType,omitempty"`
+	AvailabilityZone string `json:"availabilityZone,omitempty"`
+	//+optional
```

Comment:
I believe `+optional` is a no-op. @JoelSpeed can you confirm?

I think you can test this by removing it and re-running `make generate`. If it didn't make any change in the CRD it was a no-op.